src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -62,16 +83,16 @@ protected BaseEppoClient(
     }
 
     EppoHttpClient httpClient = buildHttpClient(host, apiKey, sdkName, sdkVersion);
-    this.configurationStore = new ConfigurationStore();
-    requestor = new ConfigurationRequestor(configurationStore, httpClient);
+    this.configurationStore = new ConfigurationStore(initialConfiguration);
+    requester = new ConfigurationRequester(configurationStore, httpClient);
```

Comment:
So my IDE suggests this spelling & rename as well, and I'm fine with changing, but the reason I hadn't was to keep this in parallel with our other SDKs which all also use requestor 🤷‍♂️